s3gof3r.go

```diff
@@ -83,25 +84,56 @@ func (s3 *S3) Bucket(name string) *Bucket {
 //workers defined in Config. This method returns a channel to receive chunks as they are completed. Chunks can
 //(and likely will) be returned out of order.
 // DefaultConfig is used if c is nil
-func (b *Bucket) GetMultiple(c *Config, files []string) (*getter, error) {
+func (b *Bucket) GetMultiple(c *Config, files []string) (*getter, []error) {
```

Comment:
how do things behave getting an array of errors?  only thing I am thinking is that its not common to get back an array of them, usually its wrapped again with an overall error.